apps/grpo/qwen3_32b.yaml

```
@@ -17,13 +17,15 @@ provisioner:
 rollout_threads: 1   # Recommended to set equal to policy.num_replicas
 
 # Observability configuration
+# logging_mode: global_reduce, per_rank_reduce, per_rank_no_reduce
 metric_logging:
   wandb:
     project: "grpo-training"
     group: "grpo_exp_${oc.env:USER}"
-    reduce_across_ranks: True
+    logging_mode: "global_reduce"
```

```suggestion
    logging_mode: "global_reduce"  # "per_rank_reduce", "per_rank_no_reduce"
```

Per my above comment. Also, do they have to be in quotes??